### PR TITLE
Added EventName for logging

### PR DESCRIPTION
**Pull request by moolta**
Added EventName

### CODE REVIEW

**Comment by cgillum** [comment]
FYI, I've created an almost identical PR here: https://github.com/Azure/azure-webjobs-sdk/pull/2544 (I didn't realize one already existed). I think mine might be a bit more complete, however.

File: src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs [hunk 1]
```diff
@@ -123,7 +123,12 @@ public static class LogConstants
         /// Gets the name of the key used to store the EventId of the log message.
         /// </summary>
         public const string EventIdKey = "EventId";
-
+        
+        /// <summary>
+        /// Gets the name of the event name of the  log message.
```
Comment (brettsam) [inline src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs:128]: Looks like there's an extra space here between `the` and `log`.

File: src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs [hunk 2]
```diff
@@ -123,7 +123,12 @@ public static class LogConstants
...
+        /// </summary>
+        public const string EventName = "EventName";
```
Comment (brettsam) [inline src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs:130]: Can you change this to `EventNameKey` to match the others?

File: src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs [hunk 3]
```diff
@@ -76,7 +76,8 @@ public ApplicationInsightsLogger(TelemetryClient client, string categoryName, Ap
             {
                 [LogConstants.CategoryNameKey] = _categoryName,
                 [LogConstants.LogLevelKey] = (LogLevel?)logLevel,
-                [LogConstants.EventIdKey] = eventId.Id
+                [LogConstants.EventIdKey] = eventId.Id,
+                [LogConstants.EventName] = eventId.Name
```
Comment (brettsam) [inline src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs:80]: Can you also add `LogConstants.EventNameKey` to the `SystemScopeKeys` at the top of this class? That will prevent this from being double-logged.

**Review by brettsam — COMMENTED** [review]
A few fairly minor comments.

**Comment by brettsam** [comment]
Sorry for the delay on this @moolta -- are you able to finalize this based on my review comments? If so, we'd love to get it in.

File: src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs (hunk 3 above)
Comment (moolta) [reply src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs:80]: I think i can abondon this PR, as I @cgillum has already raised a simialr PR, @cgillum  can you please take care of these comments in your PR https://github.com/Azure/azure-webjobs-sdk/pull/2544/files

File: src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs (hunk 2 above)
Comment (moolta) [reply src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/LogConstants.cs:130]: I think i can abandon this PR, as I @cgillum has already raised a simialr PR, @cgillum  can you please take care of these comments in your PR https://github.com/Azure/azure-webjobs-sdk/pull/2544/files

File: src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs (hunk 3 above)
Comment (cgillum) [reply src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs:80]: @moolta will do. Thanks for the quick response!

**Pull request closed without merging**